src/Parser/ShadowPriest/Modules/Spells/ShadowWordPain.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class ShadowWordPain extends Module {
+  static dependencies = {
+    enemies: Enemies,
+  };
+
+  suggestions(when) {
+    const shadowWordPainUptime = this.enemies.getBuffUptime(SPELLS.SHADOW_WORD_PAIN.id) / this.owner.fightDuration;
+    when(shadowWordPainUptime).isLessThan(0.95)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(<span>Your <SpellLink id={SPELLS.SHADOW_WORD_PAIN.id} /> uptime can be improved.</span>)
```

Comment:
In general the format for suggestions is to explain what can be improved, how to improve it (although here's it probably too obvious) and optionally why it's important. I think adding the why here might help some more clueless players.